components/SlideEach/SlideEach.tsx

```diff
@@ -0,0 +1,21 @@
+import { SlideIn } from "components/SlideIn";
+import { ISlideIn } from "components/SlideIn/SlideIn";
+import React, { FC, ReactNode } from "react";
+
+interface ISlideEach extends ISlideIn {
+	children: ReactNode;
```

Comment:
@Pratham-Mishra178 
You dont need this `children` prop too! Its already being inherited from `ISlideIn`

Please try to understand what you code does!!!